tests/integration/common-operator-integ-suite.sh

```diff
@@ -58,7 +74,6 @@ initialize_variables() {
   WD=$(dirname "$0")
   WD=$(cd "${WD}" || exit; pwd)
 
-  VERSIONS_YAML_FILE=${VERSIONS_YAML_FILE:-"versions.yaml"}
```

Comment:
This is the env var.

Reply:
Ok, I'll make the change and add the env var into the test setup

Reply:
@luksa done